presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -163,6 +163,15 @@ public void testParsingError()
         assertQueryFails("SELECT foo FROM", "line 1:16: no viable alternative at input.*");
     }
 
+    @Test
+    public void testShowPartitions()
+    {
+        assertQueryFails("SHOW PARTITIONS FROM orders", "line \\S*: Table does not have partition columns: \\S+\\.orders");
+        assertQueryFails("SHOW PARTITIONS FROM orders WHERE orderkey < 10", "line \\S*: Table does not have partition columns: \\S+\\.orders");
+        assertQueryFails("SHOW PARTITIONS FROM orders WHERE invalid_column < 10", "line 1:35: Column 'invalid_column' cannot be resolved");
```

Comment:
Why does this have a specific line position but the others don't? I think it's fine to use a specific position for all of them, and that would be more consistent with the other tests in this class.